src/main/java/io/vertx/rabbitmq/RabbitMQClient.java

```diff
@@ -145,6 +157,41 @@ default void basicConsumer(String queue, Handler<AsyncResult<RabbitMQConsumer>>
   @GenIgnore(GenIgnore.PERMITTED_TYPE)
   Future<Void> basicPublish(String exchange, String routingKey, BasicProperties properties, Buffer body);
 
+  /**
+   * Publish a message. Publishing to a non-existent exchange will result in a channel-level protocol exception,
+   * which closes the channel. Invocations of Channel#basicPublish will eventually block if a resource-driven alarm is in effect.
+   *
+   * @param deliveryTagHandler callback to capture the deliveryTag for this message.  Note that this will be called synchronously in the context of the client.
+   * @see com.rabbitmq.client.Channel#basicPublish(String, String, AMQP.BasicProperties, byte[])
+   */
+  @GenIgnore(GenIgnore.PERMITTED_TYPE)
+  void basicPublish(String exchange, String routingKey, BasicProperties properties, Buffer body, @Nullable LongConsumer deliveryTagHandler, Handler<AsyncResult<Void>> resultHandler);
```

Comment:
can you replace LongConsumer by Handler<Long> so it does not have to be gen ignored ?

Reply:
Done, but it's still failing the generation because of the BasicProperties.
The other basicPublish methods are all GenIgnore.